site/_includes/partials/showcase-card.njk

```diff
@@ -38,8 +38,9 @@
         {{ 'i18n.common.view_source' | i18n(locale) }}
       </a>
     {% endif %}
-    <a class="display-flex align-center color-primary type--small fugu-app-link" href="/fugu-showcase/#{{ id }}">
-      #
+    <a class="display-none align-center color-primary type--small web-share" href="#">
```

Comment:
This should probably be a `button`.

Reply:
@matthiasrohmer Fixed, but I fail to style it. I need to essentially apply…

```css
background-color: transparent;
border: none;
````

…but I don't know how with this funny SCSS thingy we use. I tried `class="border-none background-transparent"`, but it didn't get picked up. Help wanted.